CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changes in [34.13.0](https://github.com/matrix-org/matrix-js-sdk/releases/tag/v3
 
 * Deprecate `MatrixClient.isEventSenderVerified` ([#4527](https://github.com/matrix-org/matrix-js-sdk/pull/4527)). Contributed by @florianduros.
 * Add `restoreKeybackup` to `CryptoApi`. ([#4476](https://github.com/matrix-org/matrix-js-sdk/pull/4476)). Contributed by @florianduros.
+* The MatrixRTC format which uses a membership array in the event content and the user id (without device id)
+is not anymore supported by the js-sdk from that release onwards. This old format introduced a significant race condition that the new format
+prohibits. The reason the sdk kept supporting it was just to allow for a smooth transition. Now (6 month after its introduction)
+every EC deployment should be compatible with the new format.
```

Comment:
Changelog entries are compiled into this file a release time from the PRs: you can't add it by hand (this is entry for the previous release anyway). I suggest putting this note in the PR description and we can add it to the release notes at release time.